In [5]:
import pandas as pd

plumbers = pd.read_csv("/home/jupyter-jog141/finalproject/Plumbers.csv")

In [6]:
frequency = plumbers["zip"].value_counts().max
print(frequency)

frequency2 = plumbers["zip"].value_counts().sort_values(ascending = False).reset_index()
frequency2.head(25)
best = frequency2.iloc[0][0]
print(best)


<bound method Series.max of 15642    45
15237    44
15236    40
15227    35
15120    35
         ..
16210     1
34240     1
46112     1
15043     1
16602     1
Name: zip, Length: 209, dtype: int64>
15642


**Analysis**

After analyzing the dataset, I felt as though looking at each piece of data as a zip code would give me the best classification for a neighborhood because it was more specific than the cities provided. I also used the value_counts() method to get the top frequencies as well as utilizing the mode() so I could also acquire the top 5 frequencies in case the zip codes my other team members calculate do not match up with the top zip code of my data set. However from my data I am able to conclude that the zip code 15642 contains the most plumbers.